data/upstream.yaml

```diff
@@ -167,9 +171,17 @@ glances:
     links:
       repo: https://github.com/nicolargo/glances
       bug: https://bugzilla.redhat.com/show_bug.cgi?id=1175164
+glacier-cli:
+    status: in-progress
+    links:
+        bug: https://github.com/basak/glacier-cli/pull/54
 gnome-tweak-tool:
   status: idle
   note: Python 3 is not currently supported upstream.
+gyp:
+    status: in-progress
+    links:
+        bug: https://codereview.chromium.org/1454433002
```

Comment:
The review here has been stuck for 4 months. In fact, all of awhetter's issues are without reply for a long time. And Google's Chromium project is notorious for not accepting outside contributions.
So, I wouldn't call this "in-progress". The review link is valuable, though.